CHANGELOG.rst

```diff
@@ -1,6 +1,25 @@
 Changelog
 =========
 
+3.0.0
+-----
+
+* Shamir secret sharing scheme support allows SOPS to require multiple master
+  keys to access a data key and decrypt a file. See `sops groups -help`.
+
+* Keyservice to forward access to a local master key on a socket, similar to
+  gpg-agent. See `sops keyservice --help`.
+
+* Encrypt comments by default
+
+* Support for Google Compute Platform KMS
+
+* Refactor of the store logic to separate the internal representation SOPS
+  has of files from the external representation used in JSON and YAML files
+
+* Reencoding of versions as string on sops 1.X files, may break backward
+  compatibility but will be handled automatically.
```

Comment:
It's not handled automatically, but the following error message is shown: `Error loading file metadata: Error unmarshalling input json: json: cannot unmarshal number into Go struct field Metadata.version of type string`

Reply:
What should users who see this message do?

Reply:
They have two options:

- Rotate the file with any version of SOPS `2.x.x`, which will make `Metadata.version` a string.
- Manually edit the JSON file to ensure the entry under `sops -> version` is a string and not a number.